README.md

```diff
@@ -68,26 +94,46 @@ To run Knot.x you only need Java 8.
 
 To build it you also need Maven.
 
+## Modules
+Knot.x has two modules: **knotx-core** and **knotx-example**.
+
+*Core module* contains Knot.x application without any example data / mock endpoints. It can be deployed as standalone [verticle](http://vertx.io/docs/apidocs/io/vertx/core/Verticle.html). See [configuration section](#configuration) to deploy Knot.x core module.
+
+*Example module* contains Knot.x application, example template repositories and mock services. Internally it starts three independent verticles (for Knot.x application, example template repositories and mock services). This module is perfect entry point to Knot.x world. 
+
 ## Building
 
-To build it simply checkout the project, run:
+To build it simply checkout the project, run (from parent directory):
 
 ```
-mvn clean install
+mvn clean package
```

Comment:
It must be install, otherwise if you build it first time it fails becuase local maven repository will not have knotx-core depenency while building example module.
Remember that install phase also have package within it